geotrek/feedback/models.py

```diff
@@ -236,7 +235,10 @@ def comment_text(self):
         return html.unescape(self.comment)
 
     def send_report_to_managers(self, template_name="feedback/report_email.txt"):
-        subject = _("Feedback from {email}").format(email=self.email)
+        if self.email:
+            subject = _("Feedback from {email}").format(email=self.email)
```

Comment:
pour respecter la maniere de mettre des variables dans les trads, utilise la notation `%()s`

output = _("Today is %(month)s %(day)s.") % {"month": m, "day": d}